gamestonk_terminal/cryptocurrency/coinpaprika/README.md

```diff
@@ -0,0 +1,321 @@
+* --vs The currency to look against.  Defaults to "USD". Available options are: USD, BTC
+
+
+## ta <a name="ta"></a>
+
+````
+usage: ta [-d --days] [--vs]
+````
+
+Open Technical Analysis menu for loaded coin
+Loads data for technical analysis. You can specify currency vs which you want to show chart and also number of days to get data for.
+E.g. if you loaded in previous step Bitcoin and you want to see it's price vs USDin last 90 days range use `ta --vs USD --days 90`
+
+
+* -d/--days The number of days to look.  Defaults to 30 days. Maximum 365 days
+* --vs The currency to look against.  Defaults to "USD". Available options are: USD, BTC
+
+## basic <a name="basic"></a>
+
+````
+usage: basic
+````
+
+Shows basic information about loaded coin like: name, symbol, rank, type, description, platform, proof_type, contract, tags, parent
+
+
+## ps <a name="ps"></a>
+
+````
+usage: ps [--vs]
+````
+
+* --vs The currency to look against. Defaults to "USD". Available options are: [BTC,ETH,USD,EUR,PLN,KRW,GBP,CAD,JPY,RUB,TRY,NZD,AUD,CHF,UAH,HKD,SGD,NGN,PHP,MXN,BRL,THB,CLP,CNY,CZK,DKK,HUF,IDR,ILS,INR,MYR,NOK,PKR,SEK,TWD,ZAR,VND,BOB,COP,PEN,ARS,ISK]
+
+Price, supply related data for loaded coin. There you find metrics like:
+name, symbol, rank, supply, volume, ath, market cap, price change related metrics
+
+
+## mkt <a name="mkt"></a>
+
+````
+usage: mkt [--vs] [-t --top] [-s --sort] [--descend] [-l --links]
+````
+
+* --vs The currency to look against. Defaults to "USD". Available options are: [BTC,ETH,USD,EUR,PLN,KRW,GBP,CAD,JPY,RUB,TRY,NZD,AUD,CHF,UAH,HKD,SGD,NGN,PHP,MXN,BRL,THB,CLP,CNY,CZK,DKK,HUF,IDR,ILS,INR,MYR,NOK,PKR,SEK,TWD,ZAR,VND,BOB,COP,PEN,ARS,ISK]
+* -t/--top - number of markets to display. To display top 10 markets: `mkt --top 10 --sort trust_score`
+* -s/--sort - sort by given column. You can chose on from `pct_volume_share, exchange, pair, trust_score, volume,price`.
+* --descend - flag to sort in descending order (lowest first)
+* -l/--links - flag to show urls. Using this flag will add additional column with urls e.g. `mkt --links`
+
+Get all markets found for given coin.
+
+## ex <a name="ex"></a>
+
+````
+usage: ex [-t --top] [-s --sort] [--descend]
+````
+
+* -t/--top - number of exchanges to display. To display top 10 exchanges: `ex --top 10 --sort fiats`
+* -s/--sort - sort by given column. You can chose on from `id, name, adjusted_volume_24h_share, fiats`.
+* --descend - flag to sort in descending order (lowest first)
+
+Get all exchanges found for given coin.
+
+## twitter <a name="twitter"></a>
+
+````
+usage: twitter [-t --top] [-s --sort] [--descend]
+````
+
+* -t/--top - number of tweets to display. To display top 10 tweets: `twitter --top 10 --sort date`
+* -s/--sort - sort by given column. You can chose on from `date, user_name, status, retweet_count, like_count`.
+* --descend - flag to sort in descending order (lowest first)
+
+Show last 10 tweets for given coin.
```

Comment:
The description should be placed above the arguments bulletpoints.

Probably don't mention 10 in this comment, as this may vary based on value provided by user.